examples/gauging/gauging_itns.jl

```diff
@@ -7,38 +7,38 @@ using SplitApplyCombine
 
 using ITensorNetworks:
   message_tensors,
-  nested_graph_leaf_vertices,
-  belief_propagation_iteration,
   belief_propagation,
-  find_subgraph,
   vidal_gauge,
   symmetric_gauge,
   vidal_itn_canonicalness,
-  vidal_to_symmetric_gauge,
   initialize_bond_tensors,
   vidal_itn_isometries,
   norm_network,
-  edge_sequence
+  edge_sequence,
+  belief_propagation_iteration
 
 using NamedGraphs
 using NamedGraphs: add_edges!, rem_vertex!, hexagonal_lattice_graph
 using Graphs
 
 """Eager Gauging"""
-function eager_gauging(ψ::ITensorNetwork, bond_tensors::DataGraph, mts::DataGraph)
+function eager_gauging(
+  ψ::ITensorNetwork, pψψ::PartitionedGraph, bond_tensors::DataGraph, mts
+)
   isometries = vidal_itn_isometries(ψ, bond_tensors)
 
   ψ = copy(ψ)
   mts = copy(mts)
   for e in edges(ψ)
-    s1, s2 = find_subgraph((src(e), 1), mts), find_subgraph((dst(e), 1), mts)
+    vsrc, vdst = src(e), dst(e)
+    pe = which_partitionedge(pψψ, NamedEdge((vsrc, 1) => (vdst, 1)))
```

Comment:
```suggestion
    pe = partitionedge(pψψ, edgetype(pψψ)((vsrc, 1) => (vdst, 1)))
```
I think, not checked.

Reply:
Also, ideally `partitionedge` would be able to accept either an `AbstractEdge` or a `Pair` as the second argument.

Reply:
It looks like this file isn't being included in the tests, since the tests pass but it seems incorrect to use `which_partitionedge`.